packages/server/test/interop/react/__testHelpers.tsx

```diff
@@ -0,0 +1,265 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+
+/* eslint-disable @typescript-eslint/ban-types */
+import { trpcServer } from '../../___packages';
+import { routerToServerAndClientNew } from '../../___testHelpers';
+import '@testing-library/jest-dom';
+import { httpBatchLink } from '@trpc/client/src/links/httpBatchLink';
+import { splitLink } from '@trpc/client/src/links/splitLink';
+import {
+  TRPCWebSocketClient,
+  createWSClient,
+  wsLink,
+} from '@trpc/client/src/links/wsLink';
+import hash from 'hash-sum';
+import React, { ReactNode, useState } from 'react';
+import { QueryClient, QueryClientProvider, setLogger } from 'react-query';
+import { ZodError, z } from 'zod';
+import {
+  OutputWithCursor,
+  createReactQueryHooks,
+} from '../../../../react/src/createReactQueryHooks';
+import { TRPCError } from '../../../src/error/TRPCError';
+import { observable } from '../../../src/observable';
+import { subscriptionPullFactory } from '../../../src/subscription';
+
+setLogger({
+  log() {},
+  warn() {},
+  error() {},
+});
+
+type Context = {};
+export type Post = {
+  id: string;
+  title: string;
+  createdAt: number;
+};
+
+export function createAppRouter() {
```

Comment:
This is the v9 way of doing stuff.

```suggestion
/** @deprecated **/
export function createLegacyAppRouter() {
```

Reply:
Added your suggestion and renamed everything accordingly :+1: